app/models/account.rb

```diff
@@ -16,6 +16,7 @@ class Account < ApplicationRecord
                       :telephone_number, :vat_code, :vat_rate, :fiscal_code,
                       :state_region, :city, :country, :zip,
                       :primary_business, :business_category, :po_number
+  default_fields_are :org_name
```

Comment:
Is this PR only about the `description` field? or it's a more generic PR about adding `default_fields`?. If first, this change doesn't seem related to the PR. If second, Is `Account` the only model, other than `Cinstance`, we should update? Am I making too many questions?

Reply:
Well, the whole thing is to make the `description` field for `Cinstance` non-required :grimacing: 

But to make this happen a generic feature (`default_fields`) has been added.

We do need to then update the three models that use fields definitions: Account, Cinstance, User. But for Account and User we just put the required fields into `default_fields`, while Cinstance's `default_fields` have one required and one optional field.

Reply:
Ok I didn't notice the changes in user. I was wondering why only `Account` was updated.